Sources/NIOWebSocketServer/main.swift

```diff
@@ -203,16 +213,23 @@ let upgrader = WebSocketUpgrader(shouldUpgrade: { (head: HTTPRequestHead) in HTT
                                     channel.pipeline.add(handler: WebSocketTimeHandler())
                                  })
 
+private let httpHandler = HTTPHandler()
+
 let bootstrap = ServerBootstrap(group: group)
     // Specify backlog and enable SO_REUSEADDR for the server itself
     .serverChannelOption(ChannelOptions.backlog, value: 256)
     .serverChannelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
 
     // Set the handlers that are applied to the accepted Channels
     .childChannelInitializer { channel in
-        let config: HTTPUpgradeConfiguration = (upgraders: [ upgrader ], completionHandler: { _ in })
+        let config: HTTPUpgradeConfiguration = (
+                        upgraders: [ upgrader ], 
+                        completionHandler: { _ in 
+                            _ = channel.pipeline.remove(handler: httpHandler)
```

Comment:
I don’t think the return value should be suppressed here: the completion handler can handle future return values I think.

Reply:
If you know how best to do this, then tell me, I'm happy to use it